README.md

```diff
@@ -9,23 +9,32 @@ Groupie lets you treat your content as logical groups and handles change notific
 # Try it out:
 
 ```gradle
-implementation "com.xwray:groupie:2.7.0"
+implementation "com.xwray:groupie:$groupie_version"
 ```
 
 Groupie includes a module for Kotlin and Kotlin Android extensions. Never write a ViewHolder again—Kotlin generates view references and Groupie uses a generic holder. [Setup here.](#kotlin) 
 
 ```gradle
-implementation "com.xwray:groupie:2.7.0"
-implementation "com.xwray:groupie-kotlin-android-extensions:2.7.0"
+implementation "com.xwray:groupie:$groupie_version"
```

Comment:
Hah - thats sensible. Should've done it ages ago rather than manually updating it 😂